locale/en/foundation/tsc/index.md

```diff
@@ -22,26 +22,9 @@ can be found on the project's [GitHub profile](https://github.com/orgs/nodejs/pe
 
 ## Current members of the Technical Steering Committee
 
-* Brian White ([mscdex](https://github.com/mscdex))
-* Colin Ihrig ([cjihrig](https://github.com/cjihrig))
-* Fedor Indutny ([indutny](https://github.com/indutny))
-* James M Snell ([jasnell](https://github.com/jasnell))
-* Josh Gavant ([joshgav](https://github.com/joshgav))
-* Michael Dawson ([mhdawson](https://github.com/mhdawson))
-* Rod Vagg ([rvagg](https://github.com/rvagg))
-* Shigeki Ohtsu ([shigeki](https://github.com/shigeki))
-* Trevor Norris ([trevnorris](https://github.com/trevnorris))
-
-## Emeritus Members
-
-* Anna Henningsen ([addaleax](https://github.com/addaleax))
-* Ben Noordhuis ([bnoordhuis](https://github.com/bnoordhuis))
-* Bert Belder ([piscisaureus](https://github.com/piscisaureus))
-* Bryan Hughes ([nebrius](https://github.com/nebrius))
-* Chris Dickinson ([chrisdickinson](https://github.com/chrisdickinson))
-* Jeremiah Senkpiel ([Fishrock123](https://github.com/Fishrock123))
-* Myles Borins ([MylesBorins](https://github.com/MylesBorins))
+For the current list of TSC members, see the project
+[README.md](https://github.com/nodejs/node/blob/master/README.md#tsc-technical-steering-committee).
```

Comment:
Hmm, I think it would be nice to have the members listed here instead of linking to the GitHub repository.

Reply:
The advantage of the current approach is that syncing is not needed but I'm also fine with copying the list here.

Reply:
Yes I know, but the website is the primary entry point for people, who want to learn about the Node.js Foundation, not a GitHub repo. I know it has been challenging to keep information about the various bodies in sync, but it would be nice if we could work something out in the future.

It would be nice to have a separate MEMBERS.md in the repo which we could include on build.